index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@200;300;400;500;600;700&family=Playfair+Display:wght@400;500;600;700;800&family=Poppins:wght@300;400;500;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>Personal Portfolio</title>
+  </head>
+  <body>
+    <header class="header">
+      <nav class="navbar">
+        <div class="navbar-container">
+          <button class="mobile-menu">
+            <img src="images/icon_menu.svg" alt="mobile menu icon" />
+          </button>
+        </div>
+      </nav>
+    </header>
+    <section class="hero-banner">
+      <div class="hero-banner-container section-container">
+        <h1 class="hero-banner__title">
+          Hey there.<br> I’m Pablo.<br> <span>I’m a software developer</span>
+        </h1>
+        <p class="hero-banner__info">
+          I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+          don’t hesitate to contact me.
+        </p>
+        <ul class="socials">
+          <li class="socials-item"><a href="https://twitter.com/thasipablo_"><img src="images/twitter.svg" alt="twitter icon"></a></li>
+          <li class="socials-item"><a href="https://linkedin.com/in/thasipablo"><img src="images/linkedin.svg" alt="linkedin icon"></a></li>
+          <li class="socials-item"><a href="https://medium.com/@thasipablo"><img src="images/medium.svg" alt="medium icon"></a></li>
+          <li class="socials-item"><a href="https://github.com/thasipablo"><img src="images/github.svg" alt="github icon"></a></li>
+          <li class="socials-item"><a href="#"><img src="images/angellist.svg" alt="angellist icon"></a></li>
+        </ul>
+        <div class="banner-scroll-btn">
+          <button>
+            <img src="images/chevron_down.svg" alt="chevron down icon">
+          </button>
+        </div>
+      </div>
+    </section>
```

Comment:
- [ ] As demanded by project requirements, kindly align contents of the headline section using a flexbox.
![image](https://github.com/thasipablo/Personal-Portfolio/assets/94551484/76f2d46a-2b60-470f-a167-400633e14bba)